tx.go

```diff
@@ -69,31 +69,6 @@ func NewTransaction(creator *skademlia.Keypair, tag sys.Tag, payload []byte) Tra
 	return tx
 }
 
-func NewBatchTransaction(creator *skademlia.Keypair, tags []byte, payloads [][]byte) Transaction {
```

Comment:
@iwasaki-kenta Need your thoughts on this. `NewBatchTransaction` is only used by `cmd/graph`, `cmd/benchmark`, and `tx_applier_test.go`. Now it has been replaced with `Batch` payload like this:

```go
var batch Batch
batch.AddStake(Stake{})
batch.AddTransfer(Transfer{})
batch.AddContract(Contract{})
batch.AddNop()

tx := AttachSenderToTransaction(alice, NewTransaction(alice, sys.TagBatch, batch.Marshal()))
```

I feel that it's much cleaner to do this rather than calling `NewBatchTransaction`. Now that it's no longer used anywhere, I decided to remove it.

Reply:
That sounds much better :+1: Also, care to rebase once more with master? Some things were modified in parsing transactions.